source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,277 @@
+        vector>`.
+
+        **Examples:** prediction of the object on the picture (dog or cat?),
+        prediction of the email type (spam or not?)
+
+    Clustering
+        An :term:`unsupervised machine-learning problem<Unsupervised learning>`
+        of grouping :term:`feature vectors<Feature vector>` onto (usually
+        :term:`categorical<Categorical feature>`) groups without prior
+        information on them.
+
+        **Example:** A task of marking big star clusters at the space images
+
+    Continuous feature
+        A :term:`feature` with values in a domain of real numbers. Can be
+        :term:`interval<Interval feature>` or :term:`ratio<Ratio feature>`
+
+        **Synonyms:** quantitative feature, numerical feature
+
+        **Examples:** a person's height, the price on the house
+
+    Dataset
+        A collection of :term:`observations<Observation>`.
+
+    Feature
+        A particular property or quality of a real object or an event. Has a
+        defined type and domain. In machine learning tasks, feature is
+        considered as input variable independent from others.
+
+        **Synonyms:** attribute, variable, input variable
+
+    Feature vector
+        A vector that encodes information about single real object or an event.
+        Contains at least one :term:`feature`.
+
+        **Example:** A rectangle can be described by two features: its width and
+        height
+
+    Inference
+        A process of applying a :term:`trained<Training>` :term:`model` to the
+        :term:`dataset` with a goal to make predictions of :term:`responce`
+        values based on input :term:`feature vectors<Feature vector>`.
+
+        **Synonym:** prediction
+
+    Inference set
+        A :term:`dataset` (usually without :term:`responses<Response>` present)
+        used at the :term:`inference` stage.
+
+    Interval feature
+        A :term:`continuous feature` which values can be compared, added or
+        subtracted, but cannot be multiplied or divided.
+
+        **Examples:** a timeframe scale, a temperature in Celcius or Fahrenheit
+
+    Label
+        A :term:`response` with :term:`categorical<Categorical feature>` or
+        :term:`ordinal<Ordinal feature>` values. This is an output in
+        :term:`classification` and :term:`clustering` tasks.
+
+        **Example:** the spam-detection task has a binary label indicating whether
+        the email is spam or not
+
+    Model
+        An entiry which stores an information necessary to run :term:`inference`
+        on a new :term:`dataset`. Typically a result of :term:`training`
+        process.
+
+        **Example:** in linear regression algorithm, the model contains weight
+        values for each input feature and single bias value
+
+    Observation
+        A :term:`feature vector` and zero or more :term:`responses<Response>`.
+        Each of them can encode different objects or events.
```

Comment:
each of them - each of what?

```suggestion
        A :term:`feature vector` and zero or more :term:`responses <Response>`.
        Each of them can encode different objects or events.
```

Reply:
Second statement removed. Detailed description is written for feature vector.